core/src/smartcontracts/isi/triggers/set.rs

```diff
@@ -572,19 +571,16 @@ impl From<LoadedExecutable> for OptimizedExecutable {
 }
 
 /// [`Set::mod_repeats()`] error
-#[derive(Debug, Clone, thiserror::Error)]
+#[derive(Debug, Clone, thiserror::Error, displaydoc::Display)]
 pub enum ModRepeatsError {
-    /// Trigger not found error
-    #[error("Trigger with id = {0} not found")]
+    /// Trigger with id = {0} not found
     NotFound(TriggerId),
     /// Trigger repeats count overflow error
-    #[error("{0}")]
     RepeatsOverflow(#[from] RepeatsOverflowError),
 }
 
-/// Trigger repeats count overflow error
-#[derive(Debug, Copy, Clone, thiserror::Error)]
-#[error("Trigger repeats count overflow")]
+#[derive(Debug, Copy, Clone, thiserror::Error, displaydoc::Display)]
+/// Trigger repeats count overflow
```

Comment:
```suggestion
/// Trigger repeats count overflow
#[derive(Debug, Copy, Clone, thiserror::Error, displaydoc::Display)]
```

Reply:
Today I learned: derive macros actually do get doc-comments before the `#[derive(...)]` in their input